assignments/lambda-classes.js

```diff
@@ -1 +1,139 @@
 // CODE here for your Lambda Classes
+
+class Person{
+  constructor(attributes){
+  this.name = attributes.name;
```

Comment:
These should be indented by one. You can use prettier to do that automatically https://github.com/prettier/prettier-atom.